ex1/main.css

```diff
@@ -1,17 +1,17 @@
-h1{
+h1 {
```

Comment:
please put this class under the .center class in the following way:
.center h1 {
	color: blue;
}